Userland/Applications/HexEditor/HexDocument.cpp

```diff
@@ -228,7 +228,7 @@ bool HexDocumentUndoCommand::merge_with(GUI::Command const& other)
             m_old[relative_start + i] = typed_other.m_old[i];
     }
 
-    m_timestamp = Time::now_monotonic();
+    m_timestamp = Duration::now_monotonic();
```

Comment:
definitely should be a timestamp here, `UnixDateTime`/`UnixTimestamp::now_monotonic()` would be the thing!.